megamek/src/megamek/common/Building.java

```diff
@@ -266,7 +266,7 @@ public int getDepth() {
         }
 
         public String getDesc() {
-            return desc;
+            return Messages.getString(desc);
```

Comment:
It would be better to this in the constructor.

Reply:
That's the problem though, that causes an issue on class init. I may revert this change if I can figure out the real reason the board validator suddenly stopped running.

Reply:
I'd recommend swapping to a localized resources.getString then, like the ones in the AbstractDialogs. Haven't had any issues with them during unit testing nor initialization in MekHQ

Reply:
It looks like it failed beginning with Java 11, so at least something to dig into.

Reply:
Not the first oddity with Messages from that point